### PR TITLE
[#23] [UI] [iOS] As a user, I can see the animation when loading and showing surveys

### LINKED ISSUE #23: [UI] [iOS] As a user, I can see the animation when loading and showing surveys
## Why

The user can see a list of the surveys which have not been finished by the users on the home screen. The user can see the animation when loading and showing surveys.

## Acceptance criteria

~Show the skeleton animation for all the information on the home screen (including the date information and profile image) when preparing to show the data.~

Show the loading indicator when preparing to show the data.

When the user can navigate to the next/previous item by swiping the page to one side, the new item will fade in while the old one fades out.

## Design

- ~[Skeleton animation](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=34%3A178&scaling=scale-down)~
- [Design of the home screen](https://www.figma.com/proto/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=30%3A46&scaling=scale-down)

### PR DESCRIPTION
**Pull request by nmint8m**
- Close #23

## What happened 👀

In this PR, I show the animation when loading and showing surveys.

## Insight 📝

N/A

### CODE REVIEW

**Comment by markgravity** [comment]
@nmint8m I cant see your POW 🙏

**Review by thiennguyen0196 — COMMENTED** [review]
I cannot see your POW too @nmint8m 
And please also rebase your branch, currently it has conflicts 🙏

**Review by thiennguyen0196 — CHANGES_REQUESTED** [review]
@nmint8m The code style is good to me 👏 However for the UI, I see differences between Figma and your POW 🤔 I think it has these issues:
1. The home header skeleton section should have the longer part on top of the short one, and two lines of skeleton should have the same height
2. The survey list skeleton is missing a small skeleton text/box on top 
3. The color of skeleton should have higher opacity value, maybe 0.7 and 0.3 would be more accurate 🤔 

| Figma | POW |
|-|-|
| Screenshot 2023-08-24 at 10 29 46 AM | Screenshot 2023-08-24 at 10 30 26 AM |


|

**Comment by nmint8m** [comment]
@thiennguyen0196 I updated the POW

File: ios/ios/Sources/Presentation/Modules/Home/SurveyList/SurveyListSkeletonView.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    height: .medium,
+                    lines: 1
+                )
+                SkeletonText(
+                    width: geometry.size.width - 100.0,
+                    height: .medium,
+                    lines: 2
+                )
+                SkeletonText(
+                    width: geometry.size.width - 50,
```
Comment (markgravity) [inline ios/ios/Sources/Presentation/Modules/Home/SurveyList/SurveyListSkeletonView.swift:32]: Missing zero here `50.0` 🙏

**Review by thiennguyen0196 — APPROVED** [review]
Apart from @markgravity 's comments, the rest LGTM

**Comment by nmint8m** [comment]
@markgravity Please help me review this PR. Thank you.

**Review by markgravity — APPROVED** [review]

**Pull request merged**